window-stool.el

```diff
@@ -318,11 +318,11 @@ See: \"window-stool-use-overlays\""
                  (window-stool-window--advise-window-functions))))
     ;; clean up overlay stuff
     (progn (remove-overlays (point-min) (point-max) 'type 'window-stool--buffer-overlay)
-           (remove-hook 'post-command-hook (lambda () (window-stool--scroll-overlay-into-position)) t)
+           (remove-hook 'post-command-hook #'window-stool--scroll-overlay-into-position t)
            (setq window-scroll-functions
                  (remove #'window-stool--scroll-function window-scroll-functions))
            (kill-local-variable 'scroll-margin)
-           (cancel-timer window-stool-timer)
+           (cancel-function-timers #'window-stool-idle-fn)
```

Comment:
Oh TIL about `cancel-function-timers`. I had a different fix for this (that I stupidly forgot to push) which was to set the timer as buffer-local. 

One thing I'm not sure of is, if you use cancel-function-timer (turning off the mode in one of the buffers), because the variable is global, would it cancel the timer for the other buffers then?

Reply:
Good point. I'm quite sure it will remove it for other buffers too. The buffer-local solution, if it works, sounds much better.